tests/wicked/basic/sut/t07_static_routes_xml.pm

```diff
@@ -29,5 +29,8 @@ sub run {
     $self->assert_wicked_state(ping_ip => '10.0.2.2', iface => $ctx->iface());
 }
 
+sub test_flags {
+    return {always_rollback => 1};
```

Comment:
ok , but in separate PR and without doing this where we not really needed . meaning that PR should also include reset of this flag in most part of REF tests and some of SUT . I would like to have clear picture when we doing it and not "let's do it because it does not harm "

Reply:
Yes in a different PR. But I would just reset always. Only avoid it, if it is really wrong.

Reply:
ok